src/main.rs

```diff
@@ -124,6 +125,8 @@ fn main() -> Result<()> {
         .collect::<Result<Vec<_>>>()?;
     // build context
     let context = Context::from_ast(namespace, &libraries, opts.enable_debug)?;
+    compile_control::CompileControl::do_pass_default(&context)?;
+    go_insertion::GoInsertion::do_pass_default(&context)?;
```

Comment:
why is go_insertion after compile control? I would have expected it to be the other way around

Reply:
Gotta insert go holes for newly created groups.